packager/react-packager/src/SocketInterface/index.js

```diff
@@ -35,10 +35,15 @@ const SocketInterface = {
         }
       });
 
-      const sockPath = path.join(
+      let sockPath = path.join(
         tmpdir,
         'react-packager-' + hash.digest('hex')
       );
+      if (process.platform==='win32'){
+        sockPath = sockPath.replace(/^\//, '')
+        sockPath = sockPath.replace(/\//g, '-')
+        sockPath = '\\\\.\\pipe\\' + sockPath
```

Comment:
I copied this code from https://gist.github.com/domenic/2790533

as far as I understand, it creates a named pipe by converting the sockPath variable in a valid pipe name by replacing illegal characters for a named pipe (line 43-44), and then prefixes with the path for a named pipe on local machine. (line 45)